modules/microshift-manifests-removal-scenario.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-removal-scenario_{context}"]
+== Manifest removal scenario
+Use the following procedure in the data removal scenario to completely delete the resource defined in the kustomize manifests.
+
+.Procedure
+
+. Identify the manifest that needs to be placed in the `delete` subdirectories.
+. Use one of the following steps to create the `delete` subdirectories where the manifest is placed for deletion:
+* For RPM installation, create the `delete` subdirectory by running the following command:
++
+[source,terminal]
+----
+sudo mkdir -p _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/etc/microshift/manifests.d/delete` or `/etc/microshift/manifests/delete/`.
+* For OSTree deployments, create the `delete` subdirectory by running the following command:
++
+[source,terminal]
+----
+sudo mkdir -p _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/usr/lib/microshift/manifests.d/delete/*` or `/usr/lib/microshift/manifests/delete/`.
++
+[IMPORTANT]
+====
+For OSTree installation, the `delete` subdirectories are read only. Package the manifest into an RPM, then add them to the blueprint file before moving the manifest into the `delete` subdirectories.
+See link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{ocp-version}/html/running_applications/microshift-embedding-apps-tutorial#microshift-building-apps-rpms_microshift-embedding-apps-tutorial[Building the RPM package for the application] for the procedure to package the manifest into an RPM.
+See link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{ocp-version}/html/running_applications/microshift-embedding-apps-tutorial#microshift-adding-app-rpms-to-blueprint_microshift-embedding-apps-tutorial[Adding application RPMs to a blueprint] for the procedure to add an RPM to a blueprint.
+====
+. Move the manifest file into one of the `delete` subdirectories  under the configured manifests directory by running the following command:
++
+[source,terminal]
+----
+[sudo] mv _<path_of_manifests>_ _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_manifests>_` with the path of manifest that has the manifest to be deleted such as `/etc/microshift/manifests.d/` and replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/etc/microshift/manifests.d/delete` or `/usr/lib/microshift/manifests/delete/`.
+. Restart {microshift-short} by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl restart microshift
```

Comment:
ostree note: because RPM needs to be a part of blueprint, which is turned into an image/deployment, and in order to make that image/deployment active you need to reboot, this step is unnecessary for ostree systems.